porter_instances.json

```
@@ -0,0 +1,5 @@
+{
+  "mainnet": ["https://porter.nucypher.community"],
+  "tapir": ["https://porter-tapir.nucypher.community"],
+  "lynx": ["https://porter-lynx.nucypher.community"]
```

We're trying to deprecate the `.community` domain in favor of the `.io` domain.
 
 ```suggestion
   "mainnet": ["https://porter.nucypher.io"],
   "tapir": ["https://porter-tapir.nucypher.io"],
   "lynx": ["https://porter-lynx.nucypher.io"]
  ```